src/Zicht/Bundle/HtmldevBundle/DependencyInjection/Configuration.php

```diff
@@ -88,6 +88,7 @@ private function buildStyleguideConfigTree($rootNode)
                     ->fixXmlConfig('output')
                     ->children()
                         ->scalarNode('title')->defaultValue('Styleguide')->end()
+                        ->scalarNode('brand')->defaultValue('Styleguide')->end()
```

Comment:
Als `brand` nu leeg is, wordt ie default `"Styleguide"`. Dus staat er zowel links als rechts de tekst "Styleguide" in de top. Misschien default waarde `null` geven of een lege string `''`

Reply:
Goeie, ik heb helemaal over die default heengelezen 😶